redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -52,25 +52,32 @@ DatabaseServerV4Responses list(@NotNull @QueryParam("environmentId") String envi
     @Path("{nameOrCrn}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = DatabaseServerOpDescription.GET_BY_NAME_OR_CRN, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
-            nickname = "getDatabaseServer")
+            nickname = "getDatabaseServerByNameOrCrn")
     DatabaseServerV4Response getByNameOrCrn(@NotNull @QueryParam("environmentId") String environmentId, @PathParam("nameOrCrn") String nameOrCrn);
 
+    @GET
+    @Path("/crn/{crn}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = DatabaseServerOpDescription.GET_BY_CRN, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
+            nickname = "getDatabaseServerByCrn")
+    DatabaseServerV4Response getByCrn(@PathParam("crn") String crn);
```

Comment:
This endpoint looks perfect. My question, though: If we have this one, does "getByNameOrCrn" need to support looking up a server by CRN at all?

Reply:
Totally agree, I just did not want to touch that. Then I will.

Reply:
@bhavanki would it be ok for you to do that as part of another PR? I think it would make sense to change our api to GET, DELETE, PUT by crn only, as that seems to be the common currency.